Documentation/teaching/so2/assign0-kernel-api.rst

```diff
@@ -25,18 +25,6 @@ For details you can take a look at `the laboratory 2 <https://linux-kernel-labs.
 It is mandatory to start the implementation of the assignment from the code skeleton found in
```

Comment:
I would relocate  this line to "QuickStart" and rephrase it a bit, since the information is on the same topic and thus would prevent dividing it into two different sections.